test/test_sparse_csr.py

```diff
@@ -502,6 +574,274 @@ def test_consistency(self, layout, device, dtype, op):
             raise ValueError("Expected at least one sample with keepdim and/or explicit mask for reductions.")
 
 
+    @skipMeta
+    @all_sparse_compressed_layouts()
+    @dtypes(*all_types_and_complex_and(torch.half, torch.bool, torch.bfloat16))
+    def test_validate(self, layout, device, dtype):
+        for index_dtype in [torch.int32, torch.int64]:
+            for compressed_indices, plain_indices, values, size in self._generate_small_inputs(
+                    layout, device, dtype, index_dtype, enable_batched=True, enable_hybrid=True):
+                torch._validate_sparse_compressed_tensor_args(compressed_indices, plain_indices, values, size, layout)
+
+    def _generate_invalid_input(self, layout):
+
+        def shape(shape, basedim=0, blocksize=(1, 1)):
+            # Below, we define compressed and plain indices that
```

Comment:
This function is already defined above? Or are they not identical?

Reply:
Fixed via https://github.com/pytorch/pytorch/pull/79385#discussion_r907200124